### ISSUE TITLE
ID filtering

### ISSUE THREAD

**Issue by dtinth** [showdownspace/codegolfparty-live#2]
We may open the game to other people who play from online. However, for prizing purpose, we want to only count scores from those who registered at the event.

## Design

Provide a method that can be used in [data.ts](https://github.com/showdownspace/codegolfparty-live/blob/main/src/data.ts): `game.addPlayers(userIds: string[])`.

```ts
game.addPlayers([
  '13ca41aa0f4a9d10bef1c30e82dc4e273633114',
  '1bc6963922756fd0de3a5fe3afd940245099983',
  '21db555b08ab827e346e3cda347b06391725834',
  'a8e0c3725446bbd1bc8e84544ef6de5e4644834',
])
```

When `game.play()` is called, ignore results from users not in the ID list.

**Comment by wit03** [comment]
@dtinth Submitted a pull request for ID filtering, please check out #3

**Comment by dtinth** [comment]
Closed by #3.
